guardian/shortcuts.py

```diff
@@ -159,7 +170,7 @@ def get_perms_for_model(cls):
     return Permission.objects.filter(content_type=ctype)
 
 def get_users_with_perms(obj, attach_perms=False, with_superusers=False,
-        with_group_users=True):
+        with_group_users=True, direct_perms_only=False):
```

Comment:
Am beginning to think we need a -devel mailing list hosted somewhere so we can discuss general strategies involving django-guardian. What constitutes an backward incompatible change, when new releases should be made, should we have a section on the documentation (like Django) that describes all potential compatibility issues, etc. Any good mailing list providers for this purpose? I only know the big ones like Google and Yahoo. alioth.debian.org might also be a good option.